### ISSUE TITLE
Make the need of insecure environment (for commands like /bx_load) clearer

### ISSUE THREAD

**Issue by Zughy** [blockexchange/blockexchange#70]
I was using your mod (which is great!) and it took me a while to understand how I could allocate and load schematics directly from the website, as info are quite scattered. I suggest to make them available in game as well, either:
* make http commands like `/bx_allocate` visible as well even if the mod is not in the insecure enviroment list (and print an error message when trying executing them, telling the player to enable the insecure env for the mod)
* add a note in the local commands (e.g. `/bx_allocate_local`), telling players that they can use the online version if they put the mod in the insecure env list

**Comment by BuckarooBanzay** [comment]
> I was using your mod (which is great!)

Thanks :)

> took me a while to understand how I could allocate and load schematics directly from the website, as info are quite scattered.

So, the "download" instructions on the schematic page weren't any help i guess? Or asked differently: what do you think should be changed?

> make http commands like /bx_allocate visible as well even if the mod is not in the insecure enviroment list (and print an error message when trying executing them, telling the player to enable the insecure env for the mod)

Adding an info instead of removing them if the mod is offline sounds like better UX, yes :+1: 

Side-note: this mod does not use the insecure environment, only the http-api is used (which must be explicitly configured before using the mod)

**Comment by Zughy** [comment]
What I did:
1. I initially downloaded the mod from CDB
2. Clicked the website link in the CDB description
3. Enabled the mod and launched the world
4. Tried to paste a schematic following the website instructions in a schematic page (e.g. `/bx_allocate notsowow whale`)
5. Not working, checked `/help` and noticed there was no `/bx_allocate` command, just `/bx_allocate_local` with no mention about the former)

So basically I skipped the install instructions link on the schematic page because I thought "ok, I've already installed it, next point"
